projects/igniteui-angular/src/lib/grid/column-hiding.spec.ts

```diff
@@ -130,7 +132,7 @@ describe('Column Hiding UI', () => {
             grid.columns[0].hidden = true;
             expect(columnChooser.hiddenColumnsCount).toBe(1);
 
-            getCheckboxInput('Released').click();
+            GridFunctions.getCheckboxInput('Released', columnChooserElement, fix).click();
```

Comment:
In order to avoid duplicating the "GridFunctions." portion every time we call some util function, I suggest we define a constant for each such method in the top level describe, and use it throughout the tests as follows:
const getCheckboxInput = GridFunctions.getCheckboxInput;
then use it with the signature from the utils class:
getCheckboxInput('Released', columnChooserElement, fix).click();

Thoughts?

Reply:
It is good proposal but it won't work for this case because with this approach we will change the context of the execution of the function and when those functions invoke other functions from the GridFunctions class they won't exist anymore.